mne/datasets/_infant/__init__.py

```diff
@@ -0,0 +1 @@
+from .base import fetch_infant_template
```

Comment:
Snuck this in there because with latest `pytest` dev I was getting some weird error about submodule nesting... this makes `_infant` somehow appear as a more proper module so that it doesn't complain.